packages/manager/modules/models/BillingService/BillingService.class.js

```diff
@@ -129,7 +129,7 @@ export default class BillingService {
   }
 
   canHaveEngagement() {
-    return ['DEDICATED_SERVER'].includes(this.serviceType);
+    return ['DEDICATED_SERVER', 'VPS'].includes(this.serviceType);
```

Comment:
Maybe we should think of replacing this with capabilities from API as we can now know if a service can have engagement or not

Reply:
This info is now taken from the API